tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature

```diff
@@ -8,239 +8,142 @@ Feature: accept/decline shares coming from internal users
       | username |
       | user1    |
       | user2    |
+      | user3    |
     And user "user2" has logged in using the webUI
 
-  @smokeTest
-  @skip @yetToImplement
-  Scenario: Auto-accept disabled results in "Pending" shares
-    Given the setting "Automatically accept new incoming local user shares" in the section "Sharing" has been disabled
-    And user "user2" has shared folder "/simple-folder" with group "grp1"
-    And user "user2" has shared file "/testimage.jpg" with user "user1"
-    When user "user1" logs in using the webUI
-    Then folder "simple-folder (2)" should not be listed on the webUI
-    And file "testimage (2).jpg" should not be listed on the webUI
-    But folder "simple-folder" should be listed in the shared-with-you page on the webUI
-    And file "testimage.jpg" should be listed in the shared-with-you page on the webUI
-    And folder "simple-folder" should be in state "Pending" in the shared-with-you page on the webUI
-    And file "testimage.jpg" should be in state "Pending" in the shared-with-you page on the webUI
-
-  @skip @yetToImplement
-  Scenario: receive shares with same name from different users
-    Given the setting "Automatically accept new incoming local user shares" in the section "Sharing" has been disabled
-    And user "user2" has shared folder "/simple-folder" with user "user3"
-    And user "user1" has shared folder "/simple-folder" with user "user3"
-    When user "user3" logs in using the webUI
-    Then folder "simple-folder" shared by "User One" should be in state "Pending" in the shared-with-you page on the webUI
-    And folder "simple-folder" shared by "User Two" should be in state "Pending" in the shared-with-you page on the webUI
-
-  @skip @yetToImplement
-  Scenario: receive shares with same name from different users, accept one by one
-    Given the setting "Automatically accept new incoming local user shares" in the section "Sharing" has been disabled
-    And user "user2" has created folder "/simple-folder/from_user2"
-    And user "user2" has shared folder "/simple-folder" with user "user3"
-    And user "user1" has created folder "/simple-folder/from_user1"
-    And user "user1" has shared folder "/simple-folder" with user "user3"
-    And user "user3" has logged in using the webUI
-    When the user accepts share "simple-folder" offered by user "User One" using the webUI
-    And the user accepts share "simple-folder" offered by user "User Two" using the webUI
```

Comment:
Why did you delete all these scenarios? Are they already implemented in other feature files?

Reply:
yes they have been implemented already